Doc/whatsnew/3.2.rst

```diff
@@ -1060,15 +1060,15 @@ of nearly equal quantities:
 
 The :func:`~math.erf` function computes a probability integral or `Gaussian
 error function <https://en.wikipedia.org/wiki/Error_function>`_.  The
-complementary error function, :func:`~math.erfc`, is ``1 - erf(x)``:
+complementary error function, :func:`~math.erfc`, is ``1 - erf(x)``::
```

Comment:
I skipped the test because it can fail (on my machine I got  0.6826894921370859 instead of 0.682689492137086).